launch/tier4_planning_launch/config/scenario_planning/parking/freespace_planner/freespace_planner.param.yaml

```diff
@@ -8,6 +8,7 @@
     th_stopped_time_sec: 1.0
     th_stopped_velocity_mps: 0.01
     th_course_out_distance_m: 1.0
+    margin_m: 1.0
```

Comment:
```suggestion
    vehicle_shape_margin_m: 1.0
```

```suggestion
    vehicle_footprint_margin_m: 1.0
```

Reply:
IMO, `vehicle_shape_margin_m` is better rather than `vehicle_footprint_margin_m`.
@klu17 Could you also fix this? :bow:

Reply:
I'll go ahead and change it to `vehicle_footprint_margin_m`.